src/audio/copier/copier_gain.h

```diff
@@ -0,0 +1,231 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2024 Intel Corporation. All rights reserved.
+ *
+ * Author: Ievgen Ganakov <ievgen.ganakov@intel.com>
+ */
+
+#ifndef __SOF_COPIER_GAIN_H__
+#define __SOF_COPIER_GAIN_H__
+
+#include <sof/audio/buffer.h>
+#include <xtensa/tie/xt_hifi3.h>
```

Comment:
This does not compile with the Zephyr SDK. You need to use the SOF_USE_HIFI() macros, use `git grep -C 5 SOF_USE_HIFI` to find some examples.